files/build_templates/sonic_debian_extension.j2

```diff
@@ -307,13 +309,18 @@ sudo cp {{script}} $FILESYSTEM_ROOT/usr/bin/
 {% for service in installer_services.split(' ') -%}
 if [ -f {{service}} ]; then
     sudo cp {{service}} $FILESYSTEM_ROOT/etc/systemd/system/
-    sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable {{service}}
+    echo "{{service}}" | sudo tee -a $GENERATED_SERVICE_FILE
 fi
 {% endfor %}
 sudo LANG=C chroot $FILESYSTEM_ROOT fuser -km /sys || true
 sudo LANG=C chroot $FILESYSTEM_ROOT umount -lf /sys
 {% endif %}
 
+## Install Systemd generator for SONiC services
+sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/systemd-sonic-generator*.deb || \
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
```

Comment:
remove this line.

Reply:
Do you mean the line with dpkg -i or the line with apt-get?